### DISCUSSION TITLE
License questions

### DISCUSSION THREAD

**Discussion by soucosmo** [slint-ui/slint#5040] [Q&A] [answered]
Can I develop applications and charge for licenses?

For example: a barbershop system with a monthly subscription or something like that involving a monthly usage fee.

Can I make this type of app if I have to pay something for slint?

I confess that I had a lot of difficulty understanding the license part, which is why I've been using Iced.rs, but I've always been looking at this project, especially now that it has initial support for Android and iOS in the future.

**Comment by ogoffart** [comment, marked as answer]
> Can I develop applications and charge for licenses?

Yes, you can.

If the application is a desktop, mobile or web application (meant to be used on the user's computer or mobile phone), then you can use the royalty-free license, free of charge.

If your are providing hardware with a screen running the software, then you need to pay the license fees.

> I confess that I had a lot of difficulty understanding the license part

We're trying to make it as clear as possible, but maybe we can do a better job at that.
